backend/services/implementations/statisticService.ts

```diff
@@ -74,6 +74,28 @@ class StatisticService implements IStatisticService {
     return aggCursor[0]?.numSubmittedTests ?? 0;
   }
 
+  async getMeanScoreByTest(testId: string): Promise<number> {
+    const pipeline = [
+      filterTestsByTestId(testId),
+      {
+        $project: {
+          results: filterUngradedTests,
+        },
+      },
+      unwindResults,
+      {
+        $group: {
+          _id: null,
+          averageScore: { $avg: "$results.score" },
+        },
+      },
+    ];
+
+    const aggCursor = await MgTestSession.aggregate(pipeline);
+    const mean = aggCursor[0]?.averageScore ?? 0;
+    return Math.round(100 * mean) / 100;
```

Comment:
```suggestion
    return parseFloat(mean.toFixed(2));
```
can we actually write a helper function called like `roundTwoDecimals(num)` and refactor this line as well as [this line](https://github.com/uwblueprint/jump-math/blob/staging/backend/services/implementations/testSessionService.ts#L409) to use it? you can make a new `generalUtils.ts` file

Reply:
Done!